libs/dh/metering-point/feature-overview/src/lib/identity/dh-metering-point-identity.component.html

```diff
@@ -19,28 +19,23 @@
   *transloco="let t; read: 'meteringPoint.overview'"
 >
   <div class="watt-text-s">
-    <span [title]="t('meteringPointType')">
-      <ng-container *ngIf="meteringPoint?.meteringPointType; else emptyField">
-        {{ translationKeys?.meteringPointType | transloco }}
-      </ng-container>
-    </span>
+    <span [title]="t('meteringPointType')">{{
+      translationKeys?.meteringPointType | transloco | emDashFallback
+    }}</span>
 
-    (<span [title]="t('meteringMethod')">
-      <ng-container *ngIf="meteringPoint?.meteringMethod; else emptyField">{{
-        translationKeys?.meteringMethod | transloco
-      }}</ng-container></span
+    (<span [title]="t('meteringMethod')">{{
+      translationKeys?.meteringMethod | transloco | emDashFallback
+    }}</span
     >)
   </div>
 
   <h1 [title]="t('gsrnNumber')">{{ meteringPoint?.gsrnNumber }}</h1>
 
   <div class="watt-text-s">
     <span [title]="t('connectionState')">
-      <ng-container *ngIf="meteringPoint?.connectionState; else emptyField">
-        <dh-status-badge
-          [connectionState]="meteringPoint?.connectionState"
-        ></dh-status-badge>
-      </ng-container>
+      <dh-status-badge
```

Comment:
Question: before if connectionState wasn't defined a fallback value would be shown. Will this now going to be just blank and is that intentional?

Reply:
According to the `MeteringPointCimDto` interface, the `connectionState` property will always have a value. That was the reason for removing the fallback. From the code it loos like `connectionState` Input property might end up with `undefined` but that is mostly due to how the `meteringPoint` property is typed.

Considering all of this, do you think it's still relevant to display a fallback?

Reply:
No I think it's totally fine :)